### PR TITLE
fix 9402

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description

insert control in dialog code
## List of changes proposed in this pull request

- insert control in dialog code

## How to test

./mvnw clean test -Dcucumber.filter.tags=@TA_PFvisualizzaCodiceDelega

- [ ] Does this PR require a manual test in AWS CodeBuild?

### CODE REVIEW

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java [hunk 1]
```diff
@@ -210,11 +210,25 @@ public boolean siVisualizzaUnaDelegaConNomeDelegato(String nome, String cognome)
     }
 
     public String vericaStatoDelega() {
-        By statoDelegaBy = By.xpath("//div[@id='chip-status-success']/span");
-        this.getWebDriverWait(30).withMessage("Non si trova nessuno stato delega").until(ExpectedConditions.visibilityOfElementLocated(statoDelegaBy));
+        By statoDelegaBy = By.id("chip-status-success");
+        getWebDriverWait(30).withMessage("Non si trova nessuno stato delega").until(ExpectedConditions.visibilityOfElementLocated(statoDelegaBy));
```
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java]: preferably use **10 seconds**

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java [hunk 2]
```diff
@@ -210,11 +210,25 @@ public boolean siVisualizzaUnaDelegaConNomeDelegato(String nome, String cognome)
...
         WebElement statoDelega = this.element(statoDelegaBy);
         return statoDelega.getText();
     }
 
 
+    public void checkModaleMostraCodice() {
+        try {
+            By titoloModale = By.id("dialog-title");
+            By sottotitoloModale = By.id("dialog-description");
+            By codiceVerifica = By.xpath("//div[@data-testid='codeInput(0)']");
```
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java]: You're taking only the first input of the code, please verify all 5

**Review by afus0 — CHANGES_REQUESTED** [review]

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java (hunk 1 above)
Comment (leleOFA) [reply src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java]: done

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java (hunk 2 above)
Comment (leleOFA) [reply src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java]: done

**Review by afus0 — APPROVED** [review]

**Pull request merged**